compliance/collection/main.go

```diff
@@ -181,6 +183,47 @@ func initializeStream(ctx context.Context, cli sensor.ComplianceServiceClient) (
 	return client, config, nil
 }
 
+func manageNodescanLoop(ctx context.Context, cli sensor.ComplianceServiceClient) {
+	client, _, err := initializeStream(ctx, cli)
+	if err != nil {
+		log.Fatalf("error initializing stream to sensor: %v", err)
+	}
+	t := time.NewTicker(env.NodeScanInterval.DurationSetting())
+	scanner := nodescanv2.FakeNodeScanner{} // FIXME: Replace with real scanner (ROX-12971)
+
+	// send scan result once at startup, then every NodeScanInterval
+	if err := scanNode(client, &scanner); err != nil {
+		log.Errorf("error running scanNode: %v", err)
+	}
+
+	log.Infof("Node Scan interval: %v", env.NodeScanInterval.DurationSetting())
```

Comment:
I would move this log message a bit up:

```suggestion
	log.Infof("Node Scan interval: %v", env.NodeScanInterval.DurationSetting())
	
	// send scan result once at startup, then every NodeScanInterval
	if err := scanNode(client, &scanner); err != nil {
		log.Errorf("error running scanNode: %v", err)
	}
```